lib/pos_blockchain.go

```diff
@@ -129,42 +208,17 @@ func (bc *Blockchain) processBlockPoS(block *MsgDeSoBlock, currentView uint64, v
 	// 2. Validate QC
 	if err = bc.validateQC(block, validatorsByStake); err != nil {
```

Comment:
Similar naming suggestion here since the QC validation doesn't have side effects
```suggestion
	if err = bc.isValidPoSQuorumCertificate(block, validatorsByStake); err != nil {
```